.github/actions/test-template-action/action.yml

```diff
@@ -59,6 +59,10 @@ runs:
       shell: bash
       run: pnpm ${{ inputs.typecheck-command }}
 
+    - name: Running integration tests
```

Comment:
We are not currently running the starter templates integration tests and I wanted to include that step in the CI checks, first of all to verify the problem we have when we install the templates, however these tests do not fail because the `headers-polyfill` dependency we install in this repository is a _good_ on (we have it locked in the dependencies lock file), so I'm not sure if it's actually worthy to include this step.